### ISSUE TITLE
How to add release note

### ISSUE THREAD

**Issue by rverma-nikiai** [dependabot/dependabot-core#1330]
Can we add release note comment within pull request with the dependencies updates as we as the directory/module in which we are updating the dependencies?

**Comment by greysteil** [comment]
Could you rephrase the question? I'm not sure I understand it.

If you're asking where Dependabot gets the data for its `Release notes` tabs, it's form the release notes on GitHub for that project.
